internal/mailer/accessor.go

```diff
@@ -14,9 +14,9 @@ import (
 const (
 	insertEmailStatus = `
 		INSERT INTO email_status
-			(id, email_to, status, modified_date)
+			(id, email_to, status, vendor_id, date_sent, modified_date)
 		VALUES
-			(:id, :email_to, :status, :modified_date)
+			(:id, :email_to, :status, :vendor_id, :date_sent, :modified_date)
 	`
 	updateEmailStatus = `
 		UPDATE email_status
```

Comment:
Since you add new columns to the email_status db, you must also add that new columns in the RETURNING columns in updateEmailStatus, and make sure this endpoint works well too.

Reply:
Also update the query so it can update the new columns